src/black/__init__.py

```diff
@@ -6227,8 +6234,8 @@ def gen_python_files(
         if normalized_path is None:
             continue
 
-        # First ignore files matching .gitignore
-        if gitignore.match_file(normalized_path):
+        # Ignore files matching .gitignore if no ex
```

Comment:
ex? You mean "exclude"?

Reply:
I think I had more there. I guess it never got saved.